Automattic/woocommerce-connect-client
# Store the merchant address so it's prefilled the next times the label printing dialog is opened

Fixes #434 

To test it:
* Open the label printing dialog.
* Input a complete origin address.
* Click the "Next" button.
* Reload the page and open the label printing dialog again.

The second time you open the label printing dialog, the Origin step should be pre-filled, and if it's "valid" for the server (remember, only `Hawk St` addresses are acceptable for now) it will auto-skip the step entirely.

Ideally, the merchant address should only be "remembered" if it was used to actually purchase a label, but since the integration between the merchant host and the WCC server isn't implemented yet, that's not possible.

@jeffstieler @nabsul @allendav
